### PR TITLE
Add il state

### LINKED ISSUE #532: fetch il/state
[learn our pipeline: fetch](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#fetch)

Fetch data from: https://coronavirus.illinois.gov/s/vaccination-location

Put your script in a file named: `il/state/fetch.sh` (or `fetch.sh`, etc.)

Store it (without processing) in a new file created in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the fetch stage and how to get set up for development!

### Tips

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest fetch il/state
    ```

### LINKED ISSUE #535: parse il/state
[learn our pipeline: parse](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#parse)

Blocked by #532

Parse existing data into [`.ndjson`](http://ndjson.org/).

Read all files in the directory passed as the second argument (`sys.argv[2]`), convert them to `.ndjson`, and then output them to new files in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the parse stage and how to get set up for development!

### Tips

1. Fetch data for this site before you start developing:
    ```sh
    poetry run vaccine-feed-ingest fetch il/state
    ```

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest parse il/state
    ```

### Example
[Fetched files for `md/arcgis`](https://github.com/CAVaccineInventory/vaccine-feed-ingest-results/tree/main/md/arcgis/raw) are converted to [parsed files](https://github.com/CAVaccineInventory/vaccine-feed-ingest-results/tree/main/md/arcgis/parsed)

### LINKED ISSUE #538: normalize il/state
[learn our pipeline: normalize](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#normalize)

Blocked by #535

Normalize existing [`.ndjson`](http://ndjson.org/) records into [the Vaccinate The States data schema](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Normalized-Location-Schema).

Read all files in the directory passed as the second argument (`sys.argv[2]`), transform each record into the normalized schema (filling in as much detail as possible), then output them to new files in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the normalize stage and how to get set up for development!

### Tips

1. Fetch and parse data for this site before you start developing:
    ```sh
    poetry run vaccine-feed-ingest fetch il/state
    poetry run vaccine-feed-ingest parse il/state
    ```

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest normalize il/state
    ```

### Example
[Parsed files for `ak/arcgis`](https://github.com/CAVaccineInventory/vaccine-feed-ingest-results/tree/main/ak/arcgis/parsed) are converted to [normalized files](https://github.com/CAVaccineInventory/vaccine-feed-ingest-results/tree/main/ak/arcgis/normalized)

### PR DESCRIPTION
**Pull request by MoralCode**
# Add il state

| Key Details |
|-|
| Resolves #532 Resolves #535 Resolves #538  |
State: il |
Site: state |

## Notes
pretty simple fetch of CSV data

## Data sample

```json
{"id": "il_state:adams_county_health_department", "name": "ADAMS COUNTY HEALTH DEPARTMENT", "address": {"street1": "330 VERMONT ST", "street2": null, "city": "QUINCY", "state": "IL", "zip": "62301"}, "location": {"latitude": 39.9338798522949, "longitude": -91.4105606079102}, "contact": [{"contact_type": null, "phone": null, "website": "https://www.co.adams.il.us/government/departments/health-department/covid-19-vaccination-information", "email": null, "other": null}], "languages": null, "opening_dates": null, "opening_hours": null, "availability": null, "inventory": null, "access": null, "parent_organization": null, "links": null, "notes": null, "active": null, "source": {"source": "il_state", "id": "adams_county_health_department", "fetched_from_uri": "https://coronavirus.illinois.gov/content/dam/soi/en/web/coronavirus/documents/vaccination-locations.csv", "fetched_at": "2021-06-22T15:26:35.847235", "published_at": null, "data": {"name": "ADAMS COUNTY HEALTH DEPARTMENT", "loc_type": "Public health provider - public health clinic", "address": "330 VERMONT ST", "city": "QUINCY", "zip": "62301", "county": "ADAMS", "website": "https://www.co.adams.il.us/government/departments/health-department/covid-19-vaccination-information", "lat": "39.9338798522949", "long": "-91.4105606079102", "pindetails": "<strong>Public health provider - public health clinic</strong><br/><a href='https://www.co.adams.il.us/government/departments/health-department/covid-19-vaccination-information'>ADAMS COUNTY HEALTH DEPARTMENT</a><br/>330 VERMONT ST<br/>QUINCY, IL  62301"}}}
{"id": "il_state:advocate_aurora_south_suburban_hospital", "name": "Advocate Aurora South Suburban Hospital", "address": {"street1": "17800 South Kedzie Avenue", "street2": null, "city": "Hazel Crest", "state": "IL", "zip": "60429"}, "location": {"latitude": 41.566781, "longitude": -87.694445}, "contact": [{"contact_type": null, "phone": null, "website": "https://www.advocateaurorahealth.org/coronavirus-disease-2019/covid19-vaccine/", "email": null, "other": null}], "languages": null, "opening_dates": null, "opening_hours": null, "availability": null, "inventory": null, "access": null, "parent_organization": null, "links": null, "notes": null, "active": null, "source": {"source": "il_state", "id": "advocate_aurora_south_suburban_hospital", "fetched_from_uri": "https://coronavirus.illinois.gov/content/dam/soi/en/web/coronavirus/documents/vaccination-locations.csv", "fetched_at": "2021-06-22T15:26:35.847235", "published_at": null, "data": {"name": "Advocate Aurora South Suburban Hospital", "loc_type": "Hospital", "address": "17800 South Kedzie Avenue", "city": "Hazel Crest", "zip": "60429", "county": "Cook", "website": "https://www.advocateaurorahealth.org/coronavirus-disease-2019/covid19-vaccine/", "lat": "41.566781", "long": "-87.694445", "pindetails": "<strong>Hospital</strong><br/><a href='https://www.advocateaurorahealth.org/coronavirus-disease-2019/covid19-vaccine/'>Advocate Aurora South Suburban Hospital</a><br/>17800 South Kedzie Avenue<br/>Hazel Crest, IL  60429"}}}
{"id": "il_state:advocate_christ_medical_center", "name": "Advocate Christ Medical Center", "address": {"street1": "5220 W. 105th St\nEntrance #1", "street2": null, "city": "Oak Lawn", "state": "IL", "zip": "60453"}, "location": {"latitude": 41.70189285, "longitude": -87.74057007}, "contact": [{"contact_type": null, "phone": null, "website": "https://www.advocateaurorahealth.org/coronavirus-disease-2019/covid19-vaccine/", "email": null, "other": null}], "languages": null, "opening_dates": null, "opening_hours": null, "availability": null, "inventory": null, "access": null, "parent_organization": null, "links": null, "notes": null, "active": null, "source": {"source": "il_state", "id": "advocate_christ_medical_center", "fetched_from_uri": "https://coronavirus.illinois.gov/content/dam/soi/en/web/coronavirus/documents/vaccination-locations.csv", "fetched_at": "2021-06-22T15:26:35.847235", "published_at": null, "data": {"name": "Advocate Christ Medical Center", "loc_type": "Hospital", "address": "5220 W. 105th St\nEntrance #1", "city": "Oak Lawn", "zip": "60453", "county": "Cook", "website": "https://www.advocateaurorahealth.org/coronavirus-disease-2019/covid19-vaccine/", "lat": "41.70189285", "long": "-87.74057007", "pindetails": "<strong>Hospital</strong><br/><a href='https://www.advocateaurorahealth.org/coronavirus-disease-2019/covid19-vaccine/'>Advocate Christ Medical Center</a><br/>5220 W. 105th St\nEntrance #1<br/>Oak Lawn, IL  60453"}}}
```

## Before Opening a PR
- [X] I tested this using the CLI (e.g., `poetry run vaccine-feed-ingest <state>/<site>`)
- [X] I ran auto-formatting: `poetry run tox -e lint-fix`

### CODE REVIEW

**Comment by MoralCode** [comment]
looks like all the ID's are falling back to `unknown` which should not happen

File: vaccine_feed_ingest/runners/il/state/normalize.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+def _get_address(site):
+    return schema.Address(
+        street1=site.get("address"),
+        zip=site.get("zip"),
+        state="IL",
+        city=site.get("city"),
+    )
+
+
+# {"name": "ADAMS COUNTY HEALTH DEPARTMENT", "loc_type": "Public health provider - public health clinic", "address": "330 VERMONT ST", "city": "QUINCY", "zip": "62301", "county": "ADAMS", "website": "https://www.co.adams.il.us/government/departments/health-department/covid-19-vaccination-information", "lat": "39.9338798522949", "long": "-91.4105606079102", "pindetails": "<strong>Public health provider - public health clinic</strong><br/><a href='https://www.co.adams.il.us/government/departments/health-department/covid-19-vaccination-information'>ADAMS COUNTY HEALTH DEPARTMENT</a><br/>330 VERMONT ST<br/>QUINCY, IL  62301"}
```
Comment (bryanculbertson) [inline vaccine_feed_ingest/runners/il/state/normalize.py:76]: Remove this, or document why this is here

**Review by bryanculbertson — CHANGES_REQUESTED** [review]

File: vaccine_feed_ingest/runners/il/state/normalize.py (hunk 1 above)
Comment (MoralCode) [reply vaccine_feed_ingest/runners/il/state/normalize.py:76]: my bad, this was a dev comment to more easiily compare what fields were available

**Comment by eliblock** [comment]
Closing all prs: https://blog.vaccinateca.com/vaccinate-the-states-is-winding-down

**Pull request closed without merging**